grin.toml

```diff
@@ -105,17 +102,21 @@ cuckoo_miner_async_mode = false
 
 #cuckoo_miner_plugin_dir = "target/debug/plugins"
 
+#flag whether stratum server is enabled
+enable_stratum_server = true
```

Comment:
This is 'advanced functionality', we just need to make sure this is disabled by default

Reply:
Agreed, will make sure its disabled by default in final checkin